c3web-server/src/main/scala/com/ifunsoftware/c3web/config/ActorSystemBean.scala

```diff
@@ -0,0 +1,42 @@
+package com.ifunsoftware.c3web.config
+
+import akka.actor.ActorSystem
+import com.ifunsoftware.c3web.routing._
+import com.ifunsoftware.c3web.routing.files._
+import com.ifunsoftware.c3web.routing.groups.{ GroupsRoute, GroupRoute }
+import com.ifunsoftware.c3web.routing.journal.JournalMessagesRoute
+
+/**
+ * Created by Alexander on 9/22/2015.
+ */
+
+/**
+ * Factory method for ActorSystemBean class
+ */
+object ActorSystemBean {
+  def apply(): ActorSystemBean = new ActorSystemBean()
+}
+
+/**
+ * Defines an actor system with the actors used by
+ * the spray-person application
+ */
+class ActorSystemBean {
```

Comment:
having this kind of "root" object is fine, but i'd like to propose a bit different name. 
'*Bean' is something from a Spring world. :) How about just `Application`?